newIDE/app/src/MainFrame/InAppTutorial/StartInAppTutorialDialog.js

```diff
@@ -25,6 +29,55 @@ type Props = {|
   startTutorial: (scenario: 'resume' | 'startOver' | 'start') => Promise<void>,
 |};
 
+const titleAndContentByKey = {
```

Comment:
Probably something in the future we could store in the tutorial in a translated markdown field `markdownIntroContentByLocale`?